conformance/driver.go

```diff
@@ -160,7 +160,7 @@ func (d *Driver) ExecuteTipset(bs blockstore.Blockstore, ds ds.Batching, params
 			return big.Zero(), nil
 		}
 
-		return vm.NewVM(ctx, vmopt)
```

Comment:
Are you sure this should always use the Lotus VM? I think we need a "nv-based VM selector" that constructs one VM or another based on epoch. But that's getting into the territory of FIP-0031. Just wanted to point out that _this_ here is not something we want in this manner because it prevents us from running test vectors against the FVM entirely.

Reply:
You're right, I just skipped the testing-related changes for now (motivation for this PR is to get caterpillarnet going).